src/fragmentarium/ui/fragment/Notes.css

```diff
@@ -0,0 +1,14 @@
+.Notes__header {
+  text-align: center;
+  margin-bottom: 0;
+  font-weight: bold;
+}
+
+.Notes__notes {
```

Comment:
Use lower case names for CSS classes (https://github.com/ElectronicBabylonianLiterature/ebl-frontend#htmlcss).

The old convention was to use the component name ass block, but many linters complain about upper case letters in names.